http-api/src/lib.rs

```diff
@@ -450,8 +466,20 @@ async fn history_handler(
 
     let commits = browse(reference, ctx.paths, |browser| {
         let mut result = radicle_source::commits::<PeerId>(browser, None)?;
-        let headers: Vec<Header> = result
+
+        // To create chunks that make sense we sort the commits from newest to oldest.
+        result
             .headers
+            .sort_by(|a, b| b.committer_time.cmp(&a.committer_time));
```

Comment:
Isn't it already sorted?

Reply:
According to `radicle_source::commits`, we just map the browser history to `vcs::git::Header`, did some tests and the committer times were not always sorted.
Eventually since it's almost sorted right, and to avoid resorting all commits we could avoid sorting and do it on the client side as currently.

Reply:
Hmm ok strange, just checked again the committer times over a few repos and it looked correctly sorted, have removed the sorting now, to avoid sort repos with great amount of commits.